tools/depends/target/samba-gplv3/Makefile

```diff
@@ -57,8 +57,9 @@ ifeq ($(OS),android)
 	cd $(PLATFORM); patch -p0 < ../samba_android.patch
 	cd $(PLATFORM); patch -p1 < ../samba_off64_t.patch
 endif
-ifeq ($(TARGET_PLATFORM),appletvos)
+ifeq ($(OS), ios)
```

Comment:
no_fork_and_exec patch is now applied for ios.
Why is this wanted?

Reply:
Because iOS 11 SDK doesn't allow it anymore like tvOS did in the first place

Reply:
Oh and tvOS is a subset of iOS - it's applied to both now